keps/sig-apps/4026-job-creation-timestamp-annotation/kep.yaml

```diff
@@ -0,0 +1,34 @@
+title: KEP Template
+kep-number: 4026
+authors:
+  - "@helayoty"
+owning-sig: sig-apps
+participating-sigs:
+status: implementable
+creation-date: 2023-05-22
+reviewers:
+  - "@soltysh"
+approvers:
+  - "@soltysh"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: ""
+  beta: "v1.28"
+  stable: ""
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: JobCreationAnnotation
```

Comment:
I missed that yesterday, since this is cronjob related, let's call it `CronJobCreationAnnotation`

Reply:
Updated.